_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -306,4 +367,14 @@ if {[string first "--enable-debug" ${configure.args}] > -1} {
 
 default build.dir {${configure.dir}}
 
-default build.post_args {VERBOSE=ON}
+pre-build {
+    if {${cmake.generator} eq "Ninja"} {
+        if {![tbool use_parallel_build]} {
+            # ninja builds in parallel mode by default
+            build.post_args-append -j1
+        } elseif {${build.jobs} >= 1} {
+            build.post_args-append -j${build.jobs}
+        }
```

Comment:
I don't see why this should be done specifically in a `pre-build` block as opposed to handling it the same way as the other options above. Also, if `!use_parallel_build`, `build.jobs` will already be 1, so I don't think we need the special case at all. Just always pass `-j${build.jobs}` to ninja.

This also fixes the situation where a user has explicitly configured `buildmakejobs 1` in their `macports.conf`, but Ninja builds would still use as many tasks as the CPU supports.